_inc/client/setup-wizard/feature-toggle/index.jsx

```diff
@@ -0,0 +1,155 @@
+/**
+ * External dependencies
+ */
+import { Disabled, FormToggle } from '@wordpress/components';
+import classnames from 'classnames';
+import { translate as __ } from 'i18n-calypso';
+import PropTypes from 'prop-types';
+import React, { useCallback, useEffect, useState } from 'react';
+import { connect } from 'react-redux';
+
+/**
+ * Internal dependencies
+ */
+import Button from 'components/button';
+import Gridicon from 'components/gridicon';
+
+import {
+	mapStateToFeatureToggleProps,
+	mapDispatchToFeatureToggleProps,
+} from './map-feature-to-props';
+
+import './style.scss';
+
+let FeatureToggle = props => {
+	const {
+		title,
+		details,
+		info,
+		checked,
+		configureLink,
+		upgradeLink,
+		optionsLink,
+		isPaid = false,
+		isDisabled = false,
+	} = props;
+
+	const [ windowWidth, setWindowWidth ] = useState( false );
+
+	const handleResize = useCallback( () => {
+		setWindowWidth( window.innerWidth <= 660 ? 'small' : 'large' );
+	}, [ window.innerWidth ] );
+
+	useEffect( () => {
+		handleResize(); // Call this once to make sure windowWidth is initialized
+		window.addEventListener( 'resize', handleResize );
+		return () => {
+			window.removeEventListener( 'resize', handleResize );
+		};
+	} );
+
+	function onToggleChange() {
+		if ( upgradeLink ) {
+			window.location.href = upgradeLink;
+			return;
+		}
+
+		if ( 'function' === typeof props.onToggleChange ) {
+			props.onToggleChange( checked );
+		}
+	}
+
+	let buttonContent;
+	if ( ! checked && upgradeLink ) {
+		buttonContent = (
+			<Button href={ upgradeLink } primary>
+				{ __( 'Upgrade now' ) }
+			</Button>
+		);
+	} else if ( configureLink ) {
+		buttonContent = (
+			<Button href={ configureLink } target="_blank">
+				{ __( 'Configure' ) }
+			</Button>
+		);
+	}
+
+	const largeWindow = 'large' === windowWidth;
+	const smallWindow = 'small' === windowWidth;
+
+	let infoContent;
+	if ( info ) {
+		infoContent = <p className="jp-setup-wizard-feature-toggle-info">{ info }</p>;
+	}
+
+	let optionsLinkElement;
+	if ( optionsLink ) {
+		optionsLinkElement = (
+			<a
+				href={ optionsLink }
+				target="_blank"
+				rel="noopener noreferrer"
+				className="jp-setup-wizard-view-options-link"
+			>
+				{ __( 'View options' ) }
+			</a>
```

Comment:
Should we use `ExternalLink` here?

Reply:
For this case I don't think so, because `ExternalLink` adds the `dops-external-link` CSS class and thus changes the styling in a way that does not reflect the Figma design.